Code/Engine/GameEngine/Effects/Wind/Implementation/WindVolumeComponent.cpp

```diff
@@ -8,7 +8,7 @@
 #include <Foundation/Math/Intersection.h>
 #include <GameEngine/Effects/Wind/WindVolumeComponent.h>
 
-ezSpatialData::Category ezWindVolumeComponent::SpatialDataCategory = ezSpatialData::RegisterCategory("WindVolumes");
+ezSpatialData::Category ezWindVolumeComponent::SpatialDataCategory = ezSpatialData::RegisterCategory("WindVolumes", ezSpatialData::Flags::None);
 
```

Comment:
Should they be separated by dynamic and static volumes?

Reply:
No, that only makes sense if you expect a lot of objects (like a few 1000 or so) in this category